Gems/ScriptCanvas/Code/Editor/Translation/TranslationHelper.cpp

```diff
@@ -111,7 +111,7 @@ namespace ScriptCanvasEditor::TranslationHelper
         AzToolsFramework::AssetSystemRequestBus::BroadcastResult(
             success, &AzToolsFramework::AssetSystem::AssetSystemRequest::GetAssetSafeFolders, scanFolders);
         auto fileIO = AZ::IO::FileIOBase::GetInstance();
-
+        
```

Comment:
Nit: Looks like extra whitespace is being added here

Reply:
removed